Lib/test/test_generators.py

```diff
@@ -2252,6 +2252,11 @@ def printsolution(self, x):
   ...
 SyntaxError: 'yield from' outside function
 
+>>> f=lambda: (yield from (1,2)), (yield from (3,4))
+Traceback (most recent call last):
+  ...
+SyntaxError: 'yield from' outside function
+
```

Comment:
Can you move it between `f=lambda: (yield 1),(yield 2)` and `yield from [1,2]` to keep the lambda test cases close to each other?

Reply:
Done :)